subprojects/solver/solver-z3-legacy/README.md

```diff
@@ -0,0 +1,2 @@
+This project wraps the legacy [Z3 solver](https://github.com/Z3Prover/z3) into our common interfaces for solvers (located in the [solver](../solver) project).
+Normally, only the factory class should be used from this project to instantiate a new solver and then the common interfaces should be preferred.
```

Comment:
I think this description should be updated with information on how to produce the shipped `dll` and `so` files. As far as I understand, you had to recompile the old (`4.5.0`) version of Z3 from source, so at least this should be mentioned. Also, the exact version (Git hash), any modification you had to make. Or at the very least, it should be made clear, that downloading the binary from Z3's release page will not work with Theta.

Reply:
Sure, I'll do that.